e2e/playwright/fixtures/cmdBarFixture.ts

```diff
@@ -224,14 +224,59 @@ export class CmdBarFixture {
     // Create a handler function that saves request bodies to a file
     const requestHandler = (route: Route, request: Request) => {
       try {
-        const requestBody = request.postDataJSON()
+        // Get the raw post data
+        const postData = request.postData()
+        if (!postData) {
+          console.error('No post data found in request')
+          return
+        }
+
+        // Extract all parts from the multipart form data
+        const boundary = postData.match(/------WebKitFormBoundary[^\r\n]*/)?.[0]
+        if (!boundary) {
+          console.error('Could not find form boundary')
+          return
+        }
+
+        const parts = postData.split(boundary).filter((part) => part.trim())
+        const files: Record<string, string> = {}
+        let eventData = null
+
+        for (const part of parts) {
+          // Skip the final boundary marker
+          if (part.startsWith('--')) continue
+
+          const nameMatch = part.match(/name="([^"]+)"/)
+          if (!nameMatch) continue
+
+          const name = nameMatch[1]
+          const content = part.split(/\r?\n\r?\n/)[1]?.trim()
+          if (!content) continue
+
+          if (name === 'event') {
+            eventData = JSON.parse(content)
+          } else {
+            files[name] = content
+          }
+        }
+
+        if (!eventData) {
+          console.error('Could not find event JSON in multipart form data')
+          return
+        }
+
+        const requestBody = {
+          ...eventData,
+          files,
+        }
```

Comment:
needed a little more massaging than the last endpoint this this take multi-part data.